### PR TITLE
Update website links on README.md

### PR DESCRIPTION
**Pull request by nylira**
Changed the blog links to Medium.
Updated Tendermint website links.

### CODE REVIEW

**Comment by odeke-em** [comment]
@nylira duplicate of PR https://github.com/tendermint/tendermint/pull/583.

File: README.md [hunk 1]
```diff
@@ -70,8 +70,7 @@ Yay open source! Please see our [contributing guidelines](https://tendermint.com
 
 ### More 
 
-* [Tendermint Blog](https://tendermint.com/blog)
-* [Cosmos Blog](https://cosmos.network/blog)
+* [Tendermint Blog](https://blog.cosmos.com/tendermint/home)
```
Comment (melekes) [inline README.md]: `https://blog.cosmos.network/tendermint/home`

Comment (melekes) [inline README.md]: https://blog.cosmos.network/tendermint/home

**Comment by melekes** [comment]
@nylira thanks!

**Pull request merged**